util/archive_handler.go

```diff
@@ -4,12 +4,14 @@ import (
 	"archive/zip"
 	"errors"
 	"io/ioutil"
+	"fmt"
 
 	"gopkg.in/yaml.v2"
 )
 
 const defaultDescriptorLocation string = "META-INF/mtad.yaml"
 const defaultDescriptorName string = "mtad.yaml"
+var defaultMaxDescriptorSize int64 = 1024 * 1024
```

Comment:
This seems strange to me... I expect this to be a constant and also be configured in another way :)